### PR TITLE
[Meta Quest Mobile] v60 update, adding Quest 3

### PR DESCRIPTION
**Pull request by lionleaf**
Edit: This comment no longer applies. Ready for merge @rpavlik :)

### CODE REVIEW

File: runtimes/meta_quest_3_mobile.json.license [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Copyright 2022, The Khronos Group Inc.
```
Comment (rpavlik) [inline runtimes/meta_quest_3_mobile.json.license]: :thinking:

File: runtimes/meta_quest_2_mobile.json.license [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Copyright 2022, The Khronos Group Inc.
```
Comment (rpavlik) [inline runtimes/meta_quest_2_mobile.json.license]: :thinking:

**Review by rpavlik — CHANGES_REQUESTED** [review]
just copyright update things

File: runtimes/meta_quest_2_mobile.json.license (hunk 2 above)
Comment (lionleaf) [reply runtimes/meta_quest_2_mobile.json.license]: Assuming you're pointing to 2022? Updated to 2024.

**Review by rpavlik — APPROVED** [review]

**Comment by rpavlik** [comment]
thanks!

**Pull request merged**